pysqa/cmd.py

```diff
@@ -83,6 +85,11 @@ def command_line(argv):
                 mode_status = True
             elif opt in ("-l", "--list"):
                 mode_list = True
+            elif opt in ("-b", "--dependency"):
```

Comment:
I am not sure if `-b` is the best choice for dependencies. But as I do not have a better suggestion either as `-d` is already in use for delete, I am fine with this.